src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -78,18 +143,55 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     result.value should equal(value)
   }
 
+  "addFileMetadata" should "add the checksum validation result if the update is for a checksum" in {
```

Comment:
Does this test check anything that's not covered by the tests on lines 61 and 88? It seems like they're better tests because they check the specific value of the result boolean, not just that any boolean is saved. Unless I'm missing something?

Reply:
No, you're right, the other two check the same argument on the same method but more specifically. I've removed this one.